src/mapgen.h

```diff
@@ -193,6 +193,9 @@ class Mapgen {
 	s16 findGroundLevel(v2s16 p2d, s16 ymin, s16 ymax);
 	s16 findLiquidSurface(v2s16 p2d, s16 ymin, s16 ymax);
 	void updateHeightmap(v3s16 nmin, v3s16 nmax);
+	void getSurfaces(v2s16 p2d, s16 ymin, s16 ymax,
+	s16 *floors, s16 *ceilings, u16 *num_floors, u16 *num_ceilings);
```

Comment:
2 more tabs

Reply:
Oops ok.

Reply:
Done with 1 tab indent to match the rest of the file.